src/Core/View.js

```diff
@@ -82,15 +114,30 @@ function _preprocessLayer(view, layer, provider) {
     // probably not the best place to do this
     if (layer.type == 'color') {
         layer.frozen = layer.frozen || false;
-        layer.visible = layer.visible == undefined ? true : layer.visible;
-        layer.opacity = layer.opacity == undefined ? 1.0 : layer.opacity;
-        layer.sequence = 0;
+        _defineLayerProperty(view, layer, 'visible', true);
+        _defineLayerProperty(view, layer, 'opacity', 1.0);
+        layer.addEventListener('layer-changed', () => view.notifyChange(0, true));
```

Comment:
I'm not sure that we want to auto-connect the property changed event(s) to `notifyChange`.

I'd leave this task to the user.